spec/views/sign_up/recovery_codes/show.html.slim_spec.rb

```diff
@@ -43,4 +21,30 @@
 
     expect(rendered).to have_css('.step-3.active')
   end
+
+  context 'when render has no dependencies' do
```

Comment:
why do we want to test what happens when there are no instance variables? Would that situation occur in the app?